File: test_cases/python/function_made_sync/before.py
import asyncio

async def fetch_data(url):
    """Asynchronous data fetching."""
    # Simulate async network request
    await asyncio.sleep(0.1)
    return f"Data from {url}"

async def process_items(items):
    """Asynchronous item processing."""
    results = []
    for item in items:
        # Process each item asynchronously
        await asyncio.sleep(0.01)
        result = item * 2
        results.append(result)
    return results

async def calculate_total(numbers):
    """Async calculation function."""
    await asyncio.sleep(0.05)
    return sum(numbers)

File: test_cases/javascript/lambda_usage/before.js
// Before: No lambda/arrow function usage
function processItems(items) {
    let results = [];
    for (let i = 0; i < items.length; i++) {
        results.push(items[i] * 2);
    }
    return results;
}

function filterData(data) {
    let filtered = [];
    for (let item of data) {
        if (item.active) {
            filtered.push(item);
        }
    }
    return filtered;
}
